.appveyor.yml

```diff
@@ -0,0 +1,46 @@
+version: 'build #{build}'
+
+environment:
+  matrix:
+    - BUILD_TYPE: 386_mingw_static
+      MINGW_PATH: C:\msys64\mingw32\bin
+      CMAKE_FLAGS: -G "MinGW Makefiles" -DBUILD_SHARED_LIBS=OFF
+      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
+
+    - BUILD_TYPE: 386_msvc_static
+      CMAKE_FLAGS: -G "Visual Studio 12 2013" -DBUILD_SHARED_LIBS=OFF
+      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2013
+
+    - BUILD_TYPE: 386_shared
+      CMAKE_FLAGS: -G "Visual Studio 12 2013"
+      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2013
+
+    - BUILD_TYPE: amd64_mingw_static
+      MINGW_PATH: C:\msys64\mingw64\bin
+      CMAKE_FLAGS: -G "MinGW Makefiles" -DBUILD_SHARED_LIBS=OFF
+      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
```

Comment:
Why you use 2015 here?

Reply:
@andlabs should we use 2015 everywhere? Since it is the runtime included in [windows-build-tools](https://www.npmjs.com/package/windows-build-tools), it would require one less runtime library to inistall
for libui-node users. Actually, both 2013 and 2015 runtimes are required.

Reply:
Hm... I just always use 2015 in my projects. But yes, should work on 2013 and 2017 too.

Reply:
I can add any configuration here, but building every configuration takes around  one minute.
And in for free appveyor do not enables concurrent builds.

Reply:
I think we can just pick what @andlabs prefer.

Reply:
compilerver.hpp has my minimum versions. Testing along the matrix of every version starting from it would be nice.

Reply:
added 2015 and 2017.
14 builds will run 20-25 minutes...

https://ci.appveyor.com/project/msink/libui/build/build%20%2321

Reply:
Well, actually it was 32 minutes.